content/en/tracing/guide/adaptive_sampling.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Adaptive sampling
+
+disable_toc: false
+private: true
+further_reading:
+- link: "/tracing/trace_pipeline/ingestion_mechanisms"
+  tag: "Documentation"
+  text: "Ingestion Mechanisms"
+- link: "/tracing/trace_pipeline/ingestion_controls"
+  tag: "Documentation"
+  text: "Ingestion Control Page"
+- link: "/tracing/guide/resource_based_sampling/"
+  tag: "Documentation"
+  text: "Resource-based sampling"
+---
+
+{{< callout url="https://www.datadoghq.com/private-beta/resource-based-sampling-adaptive-sampling/" d_target="#signupModal" btn_hidden="true" btn_hidden="false" header="Request access to the beta!" >}}
+Adaptive sampling is in Preview. To request access, complete the form.
+{{< /callout >}}
+
+## Overview
+
+Datadog Adaptive Sampling will help you capture more relevant traces while remaining close to a specific budget (ingested gigabytes). By specifying a target monthly volume and enrolling one or multiple services to adaptive sampling, ensure that the consumption of these services matches the target volume at the end of the month, while keeping visibility over all services and endpoints.
+
+## How does it work?
+Datadog supports more granular configuration of existing [sampling rules][7], allowing the configuration of different sampling rates by resource. Thanks to [Remote configuration capabilities][3], based on the observed ingested traffic, Datadog is able to manage sampling rates by service, environment and resource to ensure to map the specified budget at the end of the month, while ensuring to capture visibility over low-traffic services and endpoints.
+
+Enroll new services to adaptive sampling and manage ingested volumes from the [Datadog Ingestion control page][1] by following the steps listed below.
+
+
+## Requirements
+
+- Datadog Agent [7.53.0][2] or higher.
+- [Remote Configuration][3]  enabled for your Agent.
+- `APM Remote Configuration Write` [permissions][4]. If you don’t have these permissions, ask your Datadog admin to update your permissions from your organization settings.
+
+### Tracing library version
+
+Find below the minimum tracing library version required for the feature:
+
+| Language    | Minimum version required |
+|-------------|--------------------------|
+| Java        | [v1.34.0][5]             |
+| Go          | [v1.68.0][6]             |
+| Python      | [v.2.9.6][10]            |
+| Ruby        | [v2.0.0][11]             |
+| Node.js     | [v5.16.0][12]            |
+| .NET        | [v.2.54.0][13]           |
```

Comment:
I was actually looking for this list today and remember you had pushed a docs PR, so I came to see if it was here. Thanks!

However, some of these values start with `v` and others with`v.`
```suggestion
| Python      | [v2.9.6][10]             |
| Ruby        | [v2.0.0][11]             |
| Node.js     | [v5.16.0][12]            |
| .NET        | [v2.54.0][13]            |
```